internal/dataplane/configfetcher/kongrawstate.go

```diff
@@ -12,8 +13,10 @@ import (
 // -----------------------------------------------------------------------------
 
 // KongRawStateToKongState converts a Deck kongRawState to a KIC KongState.
-func KongRawStateToKongState(rawstate *utils.KongRawState) *kongstate.KongState {
-	kongState := &kongstate.KongState{}
+func KongRawStateToKongState(rawstate *utils.KongRawState, kongVersion semver.Version) *kongstate.KongState {
+	kongState := &kongstate.KongState{
+		KongVersion: kongVersion,
```

Comment:
Do you know when setting this field has become mandatory? Asking out of curiosity